.github/workflows/pythonapp.yml

```diff
@@ -8,8 +8,9 @@ jobs:
     runs-on: ubuntu-latest
 
     strategy:
+      fail-fast: false
       matrix:
-        python-version: ['3.6', '3.10']
```

Comment:
Some concern: It has been observed before that there are too many Github Action tests in a short period of time, resulting in the call frequency of `api.auth()` exceeding the limit. Adding more versions may make automated tests more unstable and require manual retry.

Reply:
9138a40054856cfa6612b72e4b071e321b76a77e